scripts/create_sample_worksheet.py

```diff
@@ -27,19 +28,37 @@ class SampleWorksheet:
         '_{\\text{train}}}\\text{Loss}(x, y, \mathbf w) \\ \n\min_{\mathbf w \in \mathbb{R}^d} \\text{TrainLoss}(\mathbf w)$$'
     )
 
-    def __init__(self, cl, args):
+    _NAME_REGEX = '[\s\S]{0,100}'
+    _TEXT_REGEX = '[\s\S]{0,1000}'
+
+    _FULL_UUID_REGEX = '0x[a-z0-9]{32}'
+    _PARTIAL_UUID_REGEX = '0x[a-z0-9]{6}'
+    _BUNDLE_STATE_REGEX = (
```

Comment:
Make a note where these states come from in the codebase so we can try to keep this in sync.